relays/utils/src/initialize.rs

```diff
@@ -16,40 +16,81 @@
 
 //! Relayer initialization functions.
 
-use std::io::Write;
+use std::{fmt::Display, io::Write};
```

Comment:
Why where these logging changes necessary for this PR?

Reply:
After #843 the warning is now being displayed and it looks gross with the timestamp.
I can extract these changes into a separate PR if you prefer.

Reply:
Yeah, I think this was best left for another PR. It's fine though, no need to split it out now.

Your loss though - one less green square for you 😄